adaptive_n_back/experiment.js

```diff
@@ -290,6 +291,15 @@ var update_delay_block = {
 	timing_post_trial: 0
 }
 
+var update_block_acc_block = {
```

Comment:
Why did you add this function? It seems messy. I see the problem you are fixing though - when the control_block precedes the adaptive_block, block_acc isn't set to zero. To fix this just move [this line](https://github.com/jkl071/expfactory-experiments/blob/f97c4c51a05ceebc23f273b4ad3031acac8252b8/adaptive_n_back/experiment.js#113) to [start_adaptive_block](https://github.com/jkl071/expfactory-experiments/blob/f97c4c51a05ceebc23f273b4ad3031acac8252b8/adaptive_n_back/experiment.js#335). That is called before every adaptive_block so will ensure that block_acc = 0. That way we don't need another function/block/etc.



 I had already taken care of this in the "update_delay" function. I see that the block_acc can be incorrect when you start the adaptive_block after the control_block. Just move the line "block_acc = 0" to